js/app/utils.js

```diff
@@ -365,10 +365,10 @@ function unsquash_all_window_content_updates_heavy_handedly (window) {
 
 function alignment_to_justification (align) {
     switch (align) {
-        case Gtk.Align.FILL:
```

Comment:
This will break other uses of `alignment_to_justification`, such as aligning Banner.App's subtitle. That definitely should be centered when Banner.App's halign is set to FILL.